examples/warm_up_example/warmed_up_fenda/server.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+from functools import partial
+from logging import INFO
+from typing import Any, Dict, Optional
+
+import flwr as fl
+from flwr.common.logger import log
+from flwr.common.parameter import ndarrays_to_parameters
+from flwr.common.typing import Config, Parameters
+from flwr.server.client_manager import SimpleClientManager
+from flwr.server.strategy import FedAvg
+
+from examples.models.fenda_cnn import FendaClassifier, GlobalCnn, LocalCnn
+from examples.simple_metric_aggregation import evaluate_metrics_aggregation_fn, fit_metrics_aggregation_fn
+from examples.utils.functions import make_dict_with_epochs_or_steps
+from fl4health.model_bases.fenda_base import FendaJoinMode, FendaModel
+from fl4health.reporting.fl_wanb import ServerWandBReporter
+from fl4health.server.base_server import FlServer
+from fl4health.utils.config import load_config
+from fl4health.utils.random import set_all_random_seeds
+
+
+def get_initial_model_parameters() -> Parameters:
+    # Initializing the model parameters on the server side.
+    # Currently uses the Pytorch default initialization for the model parameters.
+    initial_model = FendaModel(LocalCnn(), GlobalCnn(), FendaClassifier(FendaJoinMode.CONCATENATE))
+    return ndarrays_to_parameters([val.cpu().numpy() for _, val in initial_model.state_dict().items()])
```

Comment:
Similar to the FedProx example, I think we can make it very explicit here that we don't use the initial parameters sent by the serve by setting the parameters constructed by this function to "empty"